### ISSUE TITLE
Fix colour contrast issues on hover and focus states on PDS header cta's

### ISSUE THREAD

**Issue by itsmestevey** [ministryofjustice/moj-frontend#1963]
## What

After the Probation Show & Tell on Tuesday, 4 Nov, @euhyung, our accessibility specialist, identified a few inconsistencies in the header, which appear to have been inherited from the original DPS header build.

**Issue 1: Colour contrast issue on focus state**

On keyboard focus + hover the grey chevron doesn't have sufficient colour contrast against the yellow background. 

<img width="420" height="176" alt="Image" src="https://github.com/user-attachments/assets/6100d0c2-43c6-4757-812a-de7ba4a5651b" />

In the [design file](https://www.figma.com/design/BPKX3NhVKGPQzm6UQ4hD4W/Probation-Global-Navigation?node-id=4038-1437&t=mZkRyEoEjikpVHBs-1) and in the [PDS Figma Kit](https://www.figma.com/design/AT9lj3HbJj6lYKrZLeM8CN/PDS--Probation--Figma-Kit?m=auto&node-id=13811-30&t=KiELMC8PYLi36GXX-1) the chevron on hover should be black, see screenshot below.

<img width="112" height="60" alt="Image" src="https://github.com/user-attachments/assets/8a19f474-e360-4f40-be27-b670457138f0" />

----

**Issue 2: Colour contrast issue on hover state**

On focus + hover state, when the menu is in its open state - the grey chevron doesn't have sufficient colour contrast against the grey background. Chevron should match the underline colour.

<img width="143" height="130" alt="Image" src="https://github.com/user-attachments/assets/af0b295c-906e-4a20-a9d5-8840641dc9c6" />

It's worth noting this is also an issue for DPS too, Eu will raise this with their Connect DPS.
 
## Why

Users with colour vision deficiencies may struggle to see low-contrast elements.

## Who needs to be involved

- Interaction Designer
- Developer
- Accessibility Specialist

## Done

When the focus state colours align with the original design.
Once a hover colour has been defined for the Account and Menu components in their focussed states.

**Comment by ms-ek** [comment]
Handover notes:

* This should be fixable by adjusting the SCSS code
* After implementation it might be worth catching up with Eu again to ensure the issue is completely fixed
